components/organisms/Favorites/Favorites.module.css

```diff
@@ -0,0 +1,149 @@
+.favorites__container {
+    position: absolute;
+    height: 100%;
+    width: 100vw;
+    background-image: url(https://i.imgur.com/PSLqcjG.png);
+    background-size: 100% 100%;
+    overflow-y: scroll;
+    overflow-x: hidden;
+}
+
+.favorites__title {
+    display: flex;
+    width: 100vw;
+    height: 5rem;
+    background-color: #04dbbd;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+    font-weight: 800;
+    font-size: 1.875rem;
+    color: var(--darkBlue);
+}
+
+.favorites__section--buttons {
+    display: flex;
+    align-items: center;
+    height: 3.75rem;
+    gap: 2.5rem;
+    margin: 0 5rem 0 10rem;
+    overflow-x: hidden;
+}
+
+.favorites__button {
+    padding: 0.438rem 0.75rem;
+    margin-left: 1.25rem;
+    background-color: #e4e4e4;
```

Comment:
This color should be added to the project env colors. Or modify it by one added.